src/parser/open-api/allOfResolver.js

```diff
@@ -66,7 +65,7 @@ function resolveAllOfRecursive (obj) {
  * @return {Object} definitions object that has allOf resolved
  */
 export function resolveAllOf (obj) {
-  const clonedObj = clone(obj)
+  const clonedObj = cloneDeep(obj)
```

Comment:
Sure, I can revert, it was just a little optimisation to remove `clone` from the project :)